network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/og_metadata.html

```diff
@@ -0,0 +1,8 @@
+{% load wagtailmetadata_tags i18n %}
+
+ {% get_current_language as lang_code %}
```

Comment:
if we're already doing this somewhere else I'll happily use that instead, but I couldn't find us marking the locale we're using to generate content anywhere =S